pyproject.toml

```diff
@@ -16,7 +16,7 @@ dependencies = [
     "build>=0.7.0",
     "cachetools>=4.1",
     "click>=4.0",
-    "cookiecutter>=2.1.1,<3.0",
+    "cookiecutter>=2.1.1,<2.3",  # 2.3 onwards depends on rich, which we want to avoid
```

Comment:
I wouldn't pin cookiecutter so strictly, but instead clarify in docs that if users don't want to use `rich` they might have to downgrade their `cookiecutter` version.